casper/src/main/scala/coop/rchain/casper/Validate.scala

```diff
@@ -442,7 +443,12 @@ object Validate {
                else
                  for {
                    _ <- Log[F].warn(
-                         ignore(b, "block parents did not match estimate based on justification.")
+                         ignore(
+                           b,
+                           s"block parents ${parentHashes.map(parentHash => PrettyPrinter.buildString(parentHash)).mkString(",")} did not match estimate ${computedParentHashes
```

Comment:
This message is hard to read when looking at the code. Perhaps it would make sense to extract the variable parts of it

Reply:
maybe a helper in PrettyPrinter?
```
  def buildString(hashesh: Seq[ByteString]) =
    s"${hashesh.map(hash => PrettyPrinter.buildString(hash)).mkString(",")}"
```